src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/EditCyclePage/ReviewConfirmModal/hooks/useTokensSectionValues.ts

```diff
@@ -74,9 +70,7 @@ export const useTokensSectionValues = () => {
       }),
     )
 
-  const mintRateHasDiff = Boolean(
-    currentMintRateNum !== newMintRateNum && !onlyDiscountRateApplied,
```

Comment:
This was the main problem, looking at both the ` !onlyDiscountRateApplied` AND `current === new`. Second condition not necessary.